modules/installation/pages/install-server.adoc

```diff
@@ -12,6 +12,50 @@ endif::[]
 
 
 
+{susemgr} Server is a {suse} base product from version 4.0.
+This section describes how to install {susemgr} Server from {sls} installation media.
+It assumes you have already registered the {susemgr} product with the {scc} and have a registration code.
+
+For information on registering with {scc}, or obtaining installation media, see xref:[installation-general-requirements]
+// I can't remember how we're supposed to do xrefs. Need to check with @jcayouette --LKB
+
+[NOTE]
+====
+If you are installing {susemgr} as a proof of concept, or this is a test installation, you can follow these instructions on a virtual machine rather than in production.
+Ensure your virtual machine has enough disk space and RAM by checking the requirements at xref:[install-hardware-requirements]
+====
+
+
+
+.Procedure: Installing {susemgr} Server from a DVD image
+
+. Boot your server from the installation image, you might need to adjust the boot order in the BIOS.
+. When prompted, select [guimenu]``Installation``.
++
+image::unifiedinstaller-install.png[scaledwidth=80%]
+. In the [guimenu]``Language, Keyboard and Product Selection`` screen, check the [guimenu]``SUSE Manager Server`` checkbox, and click btn:[Next].
++
+image::unifiedinstaller-product.png[scaledwidth=80%]
+. Read and agree to the End User Licence Agreement, and click btn:[Next].
+. In the [guimenu]``Registration`` screen, check the [guimenu]``Register System via scc.suse.com`` checkbox, enter your {scc} credentials, and click btn:[Next].
+. OPTIONAL: In the [guimenu]``Add On Product`` screen, select any additional or add-on products you require, and click btn:[Next].
+. In the [guimenu]``System Role`` screen, check the [guimenu]``SUSE Manager Server`` checkbox, and click btn:[Next].
+. In the [guimenu]``Suggested Partitioning`` screen, accept the default values, or use the btn:[Guided Setup] or btn:[Expert Practitioner] options to customize your partitioning model, and click btn:[Next].
+. In the [guimenu]``Clock and Time Zone`` screen, enter your region and timezone, and click btn:[Next].
+. In the [guimenu]``Local Users`` screen, create a new user, and click btn:[Next].
+. Review the settings on the [guimenu]``Installation Settings`` screen, and once you are happy click btn:[Install].
+
+Once the installation procedure has finished, you can check that you have all the required modules by using the [command]``SUSEConnect --list-extensions`` command at a command prompt.
```

Comment:
rather ``SUSEConnect --status-text``

that will displays the installed products and modules.

``SUSEConnect --list-extensions`` returns all available modules.